### PR TITLE
U276/T663-Logo  --Maintenance/logo

### PR DESCRIPTION
**Pull request by h3nx**
branchname is Maintenance/Logo, my bad

### CODE REVIEW

File: SPLASH/src/game/states/MenuState.cpp [hunk 1]
```diff
@@ -180,8 +180,15 @@ if (m_usePercentage) {
 	if (m_joiningLobby) {
 		renderJoiningLobby();
 	}
+	ImGui::PopStyleColor();
 
-
+	if (ImGui::Begin("##LOGOWINDOW", nullptr, m_standaloneButtonflags)) {
+		static ImVec2 z(3.7, 1.0);
+		Texture& logo = Application::getInstance()->getResourceManager().getTexture("splash_logo.tga");
```
Comment (DanielFredriksson) [inline SPLASH/src/game/states/MenuState.cpp]: use m_app instead of App::getInstance

File: SPLASH/src/game/states/SplashScreenState.cpp [hunk 2]
```diff
@@ -16,6 +16,7 @@ SplashScreenState::SplashScreenState(StateStack& stack)
 	m_input = Input::GetInstance();
 	m_app = Application::getInstance();
 
+	Application::getInstance()->getResourceManager().loadTexture("splash_logo.tga");
```
Comment (DanielFredriksson) [inline SPLASH/src/game/states/SplashScreenState.cpp]: use m_app instead of App::getInstance

Comment (h3nx) [reply SPLASH/src/game/states/SplashScreenState.cpp]: fan farligt att ta kod som redan finns

**Comment by Piratkopia13** [comment]
I'm gonna add a new program icon as well, taken from the logo image. Hold merge

File: SPLASH/src/game/states/MenuState.cpp [hunk 3]
```diff
@@ -103,16 +103,12 @@ bool MenuState::render(float dt, float alpha) {
...
-#ifdef DEVELOPMENT
 	auto* imguiHandler = Application::getInstance()->getImGuiHandler();
-	Application::getInstance()->getResourceManager().loadTexture("Crosshair.tga");
-	Texture& testTexture = Application::getInstance()->getResourceManager().getTexture("Crosshair.tga");
-
-	ImGui::Begin("ImageTest");
-	ImGui::Image(imguiHandler->getTextureID(&testTexture), ImVec2(100, 100));
-	ImGui::End();
+#ifdef DEVELOPMENT
 #endif
```
Comment (tofb15) [inline SPLASH/src/game/states/MenuState.cpp]: ### Junk Define block

**Pull request merged**